pkg/kubectl/cmd/get.go

```diff
@@ -217,7 +218,7 @@ func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args [
 			return err
 		}
 		if len(infos) != 1 {
-			return fmt.Errorf("watch is only supported on individual resources and resource collections - %d resources were found", len(infos))
+			return fmt.Errorf(i18n.T("watch is only supported on individual resources and resource collections - {{.Count}} resources were found", len(infos)))
```

Comment:
Done (except for the detect part), the default is return the same string, e.g. no translation, which seems like a sane default.  Later on we can add a script to check for untranslated strings (which we're going to need to do anyway)